packages/server/tests/AuthenticationSignUp.test.ts

```diff
@@ -24,11 +27,14 @@ test.group('Authentication sign up tests group', (group) => {
         initTestUtils();
 
     group.beforeEach(async () => {
+        //Init mail.trap here to prevent sending mails to random email if .env is correctly set see test `It should fail to sign up as user with given email is already in base`
+        Mail.trap(noop);
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏👌